File: 02_vehicle/java/src/Main.java
public class Main {
    public static void main(String[] args) {
        Vehicle taxi = new Vehicle();
        Vehicle moto = new Vehicle();
        Vehicle bus = new Vehicle();

        taxi.brand = "Nissan";
        taxi.plate = "RTXB101";
        taxi.color = "Amarillo";
        taxi.model = 2022;

        moto.brand = "Yamaha";
        moto.plate = "ABC42E";
        moto.color = "Negra";
        moto.model = 2021;

        bus.brand = "Chevrolet";
        bus.plate = "RFG123";
        bus.color = "De todos los colores";
        bus.model = 2005;

        System.out.println("\nDetalles taxi");
        System.out.println(taxi.details());

        System.out.println("\nDetalles moto");
        System.out.println(moto.details());

        System.out.println("\nDetalles bus");
        System.out.println(bus.details());
    }
}

File: 02_vehicle/python/main.py
class Vehicle:
    def __init__(self, brand, plate, color, model):
        self.brand = brand
        self.plate = plate
        self.color = color
        self.model = model

    def details(self):
        return f"Brand: {self.brand}\nPlate: {self.plate}\nColor: {self.color}\nModel: {self.model}"


def main():
    taxi = Vehicle("Nissan", "RTXB101", "Amarillo", 2022)
    moto = Vehicle("Yamaha", "ABC42E", "Negra", 2021)
    bus = Vehicle("Chevrolet", "RFG123", "De todos los colores", 2005)

    print("\nDetalles taxi")
    print(taxi.details())

    print("\nDetalles moto")
    print(moto.details())

    print("\nDetalles bus")
    print(bus.details())


if __name__ == '__main__':
    main()
